nsot/util/core.py

```diff
@@ -116,6 +116,35 @@ def slugify(s):
     return s
 
 
+def slugify_interface(interface=None, device_hostname=None, name=None,
```

Comment:
To me, this feels like it may better suited to be a function on the `Interface` model, which could start a pattern of each model having its own `slugify()`. Would doing that work since this function is used in `Interface.clean_name_slug()`?

I definitely see the utility for this in the tests. Would it be too ugly to make a test-specific `slugify_interface` function that wraps `Interface.slugify()` in a clean way?

Those are my thoughts at least. This is okay as-is.